tasks/deploy.js

```diff
@@ -0,0 +1,249 @@
+        console.log(
+          `NFTExtension: ${extensions.nft ? extensions.nft.address : ""}`
+        );
+        console.log(
+          `ERC20Extension: ${
+            extensions.erc20Ext ? extensions.erc20Ext.address : ""
+          }`
+        );
+        console.log("************************");
+    } else {
+        console.log("************************");
+        console.log("no migration for network " + network);
+        console.log("************************");
+    }
+});
+
+const deployRinkebyDao = async (deployFunction, network, hardHatImports) => {
+    if (!process.env.DAO_NAME)
+        throw Error("Missing env var: DAO_NAME");
+    if (!process.env.DAO_OWNER_ADDR)
+        throw Error("Missing env var: DAO_OWNER_ADDR");
+    if (!process.env.ERC20_TOKEN_NAME)
+        throw Error("Missing env var: ERC20_TOKEN_NAME");
+    if (!process.env.ERC20_TOKEN_SYMBOL)
+        throw Error("Missing env var: ERC20_TOKEN_SYMBOL");
+    if (!process.env.ERC20_TOKEN_DECIMALS)
+        throw Error("Missing env var: ERC20_TOKEN_DECIMALS");
+  
+    return await deployDao({
+        ...hardHatImports,
+        deployFunction,
+        unitPrice: toBN(toWei("100", "finney")).toString(),
+        nbUnits: toBN("100000").toString(),
+        tokenAddr: ETH_TOKEN,
+        erc20TokenName: process.env.ERC20_TOKEN_NAME,
+        erc20TokenSymbol: process.env.ERC20_TOKEN_SYMBOL,
+        erc20TokenDecimals: process.env.ERC20_TOKEN_DECIMALS,
+        maxChunks: toBN("100000").toString(),
+        votingPeriod: 600, // 600 secs = 10 mins
+        gracePeriod: 600, // 600 secs = 10 mins
+        offchainVoting: true,
+        chainId: getNetworkDetails(network).chainId,
+        deployTestTokens: true,
+        finalize: false,
+        maxExternalTokens: 100,
+        couponCreatorAddress: process.env.COUPON_CREATOR_ADDR
+            ? process.env.COUPON_CREATOR_ADDR
+            : process.env.DAO_OWNER_ADDR,
+        daoName: process.env.DAO_NAME,
+        owner: process.env.DAO_OWNER_ADDR,
+        offchainAdmin: "0xedC10CFA90A135C41538325DD57FDB4c7b88faf7",
+    });
+};
+  
+const deployMainnetDao = async (deployFunction, network, hardHatImports) => {
+    if (!process.env.DAO_NAME)
+        throw Error("Missing env var: DAO_NAME");
+    if (!process.env.DAO_OWNER_ADDR)
+        throw Error("Missing env var: DAO_OWNER_ADDR");
+    if (!process.env.ERC20_TOKEN_NAME)
+        throw Error("Missing env var: ERC20_TOKEN_NAME");
+    if (!process.env.ERC20_TOKEN_SYMBOL)
+        throw Error("Missing env var: ERC20_TOKEN_SYMBOL");
+    if (!process.env.ERC20_TOKEN_DECIMALS)
+        throw Error("Missing env var: ERC20_TOKEN_DECIMALS");
+    if (!process.env.COUPON_CREATOR_ADDR)
+        throw Error("Missing env var: COUPON_CREATOR_ADDR");
+    if (!process.env.OFFCHAIN_ADMIN_ADDR)
+        throw Error("Missing env var: OFFCHAIN_ADMIN_ADDR");
+    if (!process.env.VOTING_PERIOD_SECONDS)
+        throw Error("Missing env var: VOTING_PERIOD_SECONDS");
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819146)
```suggestion
        {throw Error("Missing env var: VOTING_PERIOD_SECONDS");}
```